lib/rails/generators/neo4j/install/templates/neo4j_rails5.rb

```diff
@@ -0,0 +1,9 @@
+# Configure NEO4J as the ORM
+Rails.application.config.generators do |g|
+  g.orm             :neo4j
+end
+
+# Configure where the neo4j database should exist
+Neo4j::Config.use do |config|
+  config[:storage_path] = "#{Rails.application.config.root}/db/neo4j-#{Rails.env}"
```

Comment:
I think we need to create some strategy to detect or offer some alternative to maintain rails philosophy about convention over configuration.
Offer a default setup and allow the user to change it, maybe using the neo4j.yml file.